docker/injector/samples/15_airflow_ingestion_aqa.json

```diff
@@ -0,0 +1,76 @@
+{
+  "data_source_oddrn": "//airflow/host/aws.airflow.aws.com/air",
+  "items": [
+    {
+      "oddrn": "//airflow/host/aws.airflow.aws.com/flights/_!+AirFlights",
+      "name": "_!+AirFlights",
+      "version": null,
+      "description": null,
+      "owner": null,
+      "metadata": [
+        {
+          "schema_url": "http://none",
+          "metadata": {
+            "Weight rule": "downstream",
+            "Email on retry": "Yes",
+            "Email": "[airflow-s3@example.com]",
+            "Parameters": "None",
+            "Retry exponential backoff": "No",
+            "Task id": "s3_task",
+            "Do xcom push": "Yes",
+            "Pool slots": 1,
+            "Task type": "S3Operator",
+            "Shallow copy attrs": "()",
+            "Run as user": "None",
+            "Depends on past": "No",
+            "Trigger rule": "all_success",
+            "Sla": "None",
+            "Execution timeout": "None",
+            "Max retry delay": "None",
+            "Queue": "default",
+            "Task concurrency": "None",
+            "Email on failure": "Yes",
+            "Operator extra links": "()",
+            "Lineage data": "None",
+            "Subdag": "None",
+            "Autocommit": "No",
+            "Database": "None",
+            "On success callback": "None",
+            "S3 conn id": "s3_default",
+            "Retries": 0,
+            "Downstream task ids": "[second_task]",
+            "Retry delay": "0:05:00",
+            "Priority weight total": "2",
+            "Schedule interval": "10:00:00",
+            "Wait for downstream": "No",
+            "On failure callback": "None",
+            "On retry callback": "None",
+            "Pool": "default_pool",
+            "Priority weight": "2",
+            "Resources": "None"
+          }
+        }
+      ],
+      "updated_at": null,
+      "created_at": null,
+      "type": "JOB",
+      "dataset": null,
+      "data_transformer": {
+        "source_code_url": null,
+        "sql": null,
+        "inputs": [
+          "//s3/cloud/aws/account/111111111111/region/us-west-2/flights/arrives"
+        ],
+        "outputs": [
+          "//s3/cloud/aws/account/111111111111/region/us-west-2/flights/departure"
+        ]
+      },
+      "data_transformer_run": null,
+      "data_quality_test": null,
+      "data_quality_test_run": null,
+      "data_input": null,
+      "data_consumer": null,
+      "data_entity_group": null
+    }
+  ]
+}
```

Comment:
Let's get rid of all unused `*.json` files.

- In tests, we are mostly using "BookETL" (from file `docker/injector/samples/04_etl_ingestion.json`).
- We also using "_!+AirFlights" (from file `docker/injector/samples/15_airflow_ingestion_aqa.json`).

And it looks like all other .json files could be safely removed

Reply:
I think we can also consult with @DementevNikita and team on the following question:
- Could we update the name of a data object in one of the existing JSON files to cover our test case "Search entity which starts with special characters"?

For instance: can we change `"data_lake"`  into `"_data_lake"` (in following file [08_s3_ingestion.json#L6](https://github.com/opendatadiscovery/odd-platform/blob/50f9e7a/docker/injector/samples/08_s3_ingestion.json#L6))?